backend/src/test/java/org/cryptomator/hub/api/VaultResourceTest.java

```diff
@@ -596,20 +601,21 @@ public void getMembers3() {
 	@TestMethodOrder(MethodOrderer.OrderAnnotation.class)
 	public class ManageGroups {
 
-		private final String vault4AdminJWT = "eyJhbGciOiJFUzM4NCIsInR5cCI6IkpXVCIsInZhdWx0SWQiOiJ2YXVsdDQifQ.eyJpYXQiOjE1MTYyMzkwMTUsImV4cCI6NTgxNjIzOTAzMCwibmJmIjoxNTE2MjM5MDAwfQ.aSe-P9j1vybUPS_Y1PDQ5knh_slaNulX365TSJNBYGY5dooUTZicu27_h_jYC-vvsafn4sjQhifEoXJq1U5r2kEp8ZV_Bn_4GdLLAG2JDewuAvWlkqks1un4spEO8yO4";
+											// eyJ2YXVsdElkIjoiN0U1N0MwREUtMDAwMC00MDAwLTgwMDAtMDAwMTAwMDA0NDQ0IiwidHlwIjoiSldUIiwiYWxnIjoiRVMzODQifQ.eyJpYXQiOjE2NzY2NTEwMTd9.HmbkbBO5yemWECLGwp8Q_dOEW1aoENkpZms5FFLhpPjTXb6cvCddBZF0ElK-6a4eG9Z_nR6RV0xsLA8BekBAjlTfYvqRwdQE5WJ2MzMtdxBYjs1Pic52jnJ0by8ko8Ap
```

Comment:
What does this comment mean?

Reply:
oh just some left-over code from testing. I'll remove it.